hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4352,6 +4352,17 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.client.packet.recovery.max.times</name>
```

Comment:
dfs.client.pipeline.recovery.max-retries?

Reply:
Thanks @Hexiaoqiao for review, i have fixed it in latest pull request.